js/languages/en_US.json

```diff
@@ -453,13 +453,11 @@
       "sectionHeader": "Donations",
```

Comment:
The way the underline class is being applied on the hover of the button is making the button text rise a couple pixels.

Reply:
FWIW, that has nothing to do with that line of code.

Reply:
:)

I'll take a look at it.

Reply:
Still getting this. It happens on the in-active coin nav.